test/test_sdo.py

```diff
@@ -110,8 +110,7 @@ def test_block_download(self):
             (RX, b'\xa1\x00\x00\x00\x00\x00\x00\x00')
         ]
         data = b'A really really long string...'
-        with self.network[2].sdo['Writable string'].open(
-            'wb', size=len(data), block_transfer=True) as fp:
+        with self.network[2].sdo['Writable string'].open('wb', size=len(data), block_transfer=True) as fp:
```

Comment:
Do we have an agreement on line length in canopen? Personally I think it should be more than 80, so I support this change.